apps/common/controllers/add-another-address-loop.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const _ = require('lodash');
+const BaseController = require('./base');
+
+module.exports = class AddAnotherAddressController extends BaseController {
+  constructor(options) {
+    super(options);
+    if (options.locals.field) {
+      this.field = options.locals.field;
+    } else {
+      // eslint-disable-next-line no-console
+      console.error('No field set in locals');
```

Comment:
@joefitter It can't be omitted and I don't think a default value can be set. Is my console.error not acceptable?